research/object_detection/g3doc/context_rcnn.md

```diff
@@ -130,6 +134,22 @@ python object_detection/dataset_tools/context_rcnn/generate_embedding_data.py \
   --embedding_model_dir path/to/exported_model_directory/saved_model
 ```
 
+To use with Dataflow, run:
```

Comment:
You should explain at the top of the document what DataFlow is and how to set up DataFlow on cloud.

Reply:
We also need to specify that to run on dataflow you need to install apache-beam[gcp]